R/pipelines.R

```diff
@@ -519,9 +519,83 @@ pipeline_survtoregr = function(method = 1, regr_learner = lrn("regr.featureless"
   gr
 }
 
+#' @name mlr_graphs_survtoclassif_disctime
+#' @title Survival to Classification Reduction Pipeline
+#' @description Wrapper around multiple [PipeOp][mlr3pipelines::PipeOp]s to help in creation
+#' of complex survival reduction methods.
```

Comment:
@studener we need a @details section here, see `pipeline_survtoregr` for an example - pretty muchyou explain what pipeops go into the discrete time pipeline and what they do